db/migrate/20200408150738_create_notes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateNotes < ActiveRecord::Migration[6.0]
+  def change
+    create_table :notes do |t|
+      t.string :title
+      t.text :message
+      t.belongs_to :application_choice, null: false, foreign_key: true
```

Comment:
Should we add a relationship to the author (provider user) here as well?

Reply:
Good point - knowing who wrote a not is probably something we would want to know haha.